routes/user.js

```diff
@@ -207,29 +207,19 @@ router.get("/clients", (req, res) => {
   }).catch(error500(res));  
 });
 
-// TODO: If owner, show a list of all users to attribute client to
-//       Same for supervisor, but limit only within department
 router.get("/clients/create", (req, res) => {
-  res.render("clients/create");
+  res.render("clients/create", { users: null });
 });
 
 router.post("/clients/create", (req, res) => {
-  let userID = req.user.cmid;
-  let first  = req.body.first;
-  let middle = req.body.middle ? req.body.middle : "";
-  let last   = req.body.last;
-  let dob    = req.body.DOB;
-  let so     = req.body.uniqueID1 ? req.body.uniqueID1 : null;
-  let otn    = req.body.uniqueID2 ? req.body.uniqueID2 : null;
-
   Client.create(
-          userID, 
-          first, 
-          middle, 
-          last, 
-          dob, 
-          otn, 
-          so
+          req.user.cmid, 
+          req.body.first, 
+          req.body.middle ? req.body.middle : "", 
+          req.body.last, 
+          req.body.dob, 
+          req.body.uniqueID2 ? req.body.uniqueID2 : null, 
+          req.body.uniqueID1 ? req.body.uniqueID1 : null
```

Comment:
Wouldn't necessarily do this. `let so = req.bodyUniqueID1` is making it clearer that the creation function wants a param called `so`. `so` should probably also have a better name.

Defining parameters with inconsistent names into variables with correct names seems like a good step. 

Again, this is a good time to point out that we should probably switch to dictionaries over ordered params for something like "create" so that the names are present in the function call.